specification/newrelic/NewRelic.Management/examples/Monitors_GetMetricStatus_MinimumSet_Gen.json

```diff
@@ -7,12 +7,17 @@
     "resourceGroupName": "rgNewRelic",
     "monitorName": "fhcjxnxumkdlgpwanewtkdnyuz",
     "request": {
+      "azureResourceIds": [
+        "enfghpfw"
+      ],
       "userEmail": "ruxvg@xqkmdhrnoo.hlmbpm"
     }
   },
   "responses": {
     "200": {
-      "body": {}
+      "body": {
+        "azureResourceIds": []
+      }
```

Comment:
Does this suppose to be blank given the request have entry?

Reply:
What is the reason?

Reply:
There is no azureResourceIds in monitoring state.